docs/source/index.rst

```diff
@@ -81,6 +81,7 @@ Features described in this documentation are classified by release status:
    torch.autograd <autograd>
    torch.library <library>
    cuda
+   torch._awaits <_awaits>
```

Comment:
Hmm, do we really want to document internal library?

Reply:
I've added it to pass public-api doc test before converted it to be private API :) 
I will remove it for now.